src/Enhavo/Bundle/RevisionBundle/EventListener/RevisionDoctrineFilterListener.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Enhavo\Bundle\RevisionBundle\EventListener;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Enhavo\Bundle\RevisionBundle\Doctrine\RevisionFilter;
+
+class RevisionDoctrineFilterListener
+{
+    public function __construct(
+        private readonly EntityManagerInterface $entityManager,
+        private readonly array                  $configuration
```

Comment:
Just use enabled boolean, so this class doesn't depends on a configuration array.